src/js/pre-need/config/form.js

```diff
@@ -259,7 +262,7 @@ const formConfig = {
             applications: {
               items: {
                 'ui:title': claimantHeader,
-                'ui:description': <p>You aren't required to fill in <strong>all</strong> fields, but VA can evaluate your claim faster if you provide more information.</p>,
+                'ui:description': <p>You aren't required to fill in <strong>all</strong> fields, but we can review your application faster if you provide more information.</p>,
```

Comment:
I think you can use the description component here, too?

Reply:
Thanks @jbalboni, excellent idea, change made.